packages/codefixes/test/ts-codefixes.test.ts

```diff
@@ -175,6 +175,11 @@ describe('ts-codefixes', () => {
       'convertTypedefToType/failing/fail-2.ts',
       'convertTypedefToType/passing/pass-2.ts'
     );
+
+    await runTest(
+      'convertTypedefToType/failing/fail-3.ts',
+      'convertTypedefToType/passing/pass-3.ts'
```

Comment:
How do these tests differ?

Reply:
The `fail-3.ts` is incoming file, the `pass-3.ts` is output ("snapshot"). We don't expect any changes in cases like this, this is why they are the same.